corehq/tabs/tabclasses.py

```diff
@@ -545,8 +541,6 @@ def sidebar_items(self):
                 DownloadNewSmsExportView,
                 BulkDownloadFormExportView,
                 BulkDownloadNewFormExportView,
-                EditCustomFormExportView,
-                EditCustomCaseExportView,
```

Comment:
can these edit views also be removed?

Reply:
Probably, although I think I'd prefer to do that separately, since you can still conceivably access the edit view for the remaining old exports if you know the URL.  That might come in handy when inspecting what they do during the migration process.